.github/workflows/deploy-app.yml

```diff
@@ -39,9 +39,7 @@ jobs:
         key: ${{ secrets.KEY }}
         passphrase: ${{ secrets.PASSPHRASE }}
         script: |
-          become mismatch-finder rm -rf mismatch-finder-repo
-          cp -rv mismatch-finder-repo/ ~tools.mismatch-finder/
+          # Make sure ~tools.mismatch-finder/mismatch-finder-repo is group writable
+          become mismatch-finder chmod -R g+rwx ~tools.mismatch-finder/mismatch-finder-repo
+          rsync -rltD --delete --delay-updates --exclude mismatch-finder-repo/.env --exclude mismatch-finder-repo/storage/app/allowlist/uploaders.txt ~/mismatch-finder-repo ~tools.mismatch-finder/
```

Comment:
One more thing, we would probably want to keep all of `storage/app` intact for now, as we also upload files to a subdirectory there.

Reply:
Sounds good, changed.